src/basic/exec-util.c

```diff
@@ -256,7 +256,7 @@ int execute_directories(
 static int gather_environment_generate(int fd, void *arg) {
         char ***env = arg, **x, **y;
         _cleanup_fclose_ FILE *f = NULL;
-        _cleanup_strv_free_ char **new;
+        _cleanup_strv_free_ char **new = NULL;
```

Comment:
we probably should not name vaiables `new` btw... We have a macro by that name. The only reason this compiles at all is that the macro is actually called `new()` rather than just `new`, but the clash is philosophically unsexy...